api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -389,19 +390,32 @@ private User initializeUserIfNeeded() {
   }
 
   @Override
-  public ResponseEntity<Profile> submitIdVerification(IdVerificationRequest request) {
-    // TODO(dmohs): Prevent this if the user has already attempted verification?
-    Person person = blockscoreService.createPerson(
-      request.getFirstName(), request.getLastName(),
-      new Address()
-        .setStreet1(request.getStreetLine1()).setStreet2(request.getStreetLine2())
-        .setCity(request.getCity()).setSubdivision(request.getState())
-        .setPostalCode(request.getZip()).setCountryCode("US"),
-      request.getDob(),
-      request.getDocumentType(), request.getDocumentNumber()
-    );
+  public ResponseEntity<Profile> submitIdVerification() {
+    User user = userProvider.get();
+    if (user.getRequestedIdVerification() == null || user.getRequestedIdVerification() == false) {
```

Comment:
Thinking about it some more... what happens if somebody enters in some information incompletely or incorrectly, requests a review, and gets rejected... will we clear out this flag at that point so they can fix their info and request verification again?

Reply:
Yeah, my understanding is that we don't have too much process around rejection/re-approval/re-review right now.